dima/react-movies/src/components/header/header.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import "./header.css";
+interface HeaderProps {
+  moviePage: boolean;
```

Comment:
don't make props so specific. If for example you needed to show button on another page, you would add a new flag ? 

moviePage -> showSearchBtn

Reply:
fixed